MLBSpider.py

```diff
@@ -1,58 +0,0 @@
-import scrapy
-from datetime import datetime
-import datetime
-from scrapy_splash import SplashRequest
-from dateutil import tz
-import csv
-from scrapy.crawler import CrawlerProcess
-
-
-class MLBSpider(scrapy.Spider):
-    name = 'MLBSpider'
-    SPLASH_ARGS = {
-        'wait': 20.0,
-        'images': 0,
-        'render_all': 1,
-        'timeout': 40.0,
-        'resource_timeout': 40.0}
-
-    def start_requests(self):
-        url = 'http://www.espn.com/mlb/schedule'
-        yield SplashRequest(url, callback=self.parse, args=dict(self.SPLASH_ARGS))
-
-    def parse(self, response):
-
-        #Scrapes Home Team Name and appends to team list
-        teams = []
-        for team in response.xpath('//div[@class="responsive-table-wrap"][2]/table/tbody/tr/td[@class="home"]/div/a/abbr'):

```

Comment:
When a line gets this long I like to pull parts of it out for readability, and would assign the xpath to a string just above, `team_xpath = '//div[@class="responsive-table-wrap"][2]/table/tbody/tr/td[@class="home"]/div/a/abbr'`
and `for team in response.xpath(team_xpath):`

Just a style thing. You could also add this to the `xpath_map` suggested above instead.